CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 option(GOTCHA_ENABLE_TESTS "Enable internal tests" Off)
 option(GOTCHA_ENABLE_COVERAGE_TESTS "Enable coverage tests" Off)
 project(gotcha)
+
+set(LIBTOOL_INTERFACE 0)
```

Comment:
@mplegendre : for managing versioning, our new thing we have to handle is to maintain this triple appropriately as we make PRs